packages/cli/src/PublicApi/v1/handlers/workflows/workflows.service.ts

```diff
@@ -1,5 +1,5 @@
 import { FindManyOptions, In, UpdateResult } from 'typeorm';
-import { intersection } from 'lodash';
+import intersection from 'lodash.intersection';
```

Comment:
For learning, why is this better?

Reply:
mainly to only have the necessary dependencies. in this case we don't need the whole lodash package as a dependency but just `intersection`

Reply:
Thanks! I thought a full import only happened with `import _ from 'lodash'` instead of `import { something } from 'lodash'`. Good to know.

Reply:
yeah thats correct too, but mainly its about having the whole `lodash` package in node_modules or just `lodash.intersection`. But, i see we already have lodash as a dependency due to other packages depending on it, so doesn't make a difference in the end 🤷‍♂️